src/main/java/pages/InformationPage.java

```diff
@@ -0,0 +1,99 @@
+    private WebElement userFirstName ;
+
+    @FindBy(xpath = "//input[@id='checkout_shipping_address_last_name']")
+    private WebElement userLastName ;
+
+    @FindBy(xpath = "//input[@id='checkout_shipping_address_company']")
+    private WebElement userCompany;
+
+    @FindBy(xpath = "//input[@id='checkout_shipping_address_address1']")
+    private WebElement userAddress1;
+
+    @FindBy(xpath = "//input[@id='checkout_shipping_address_address2']")
+    private WebElement userAddress2;
+
+    @FindBy(xpath = "//input[@id='checkout_shipping_address_city']")
+    private WebElement userCity;
+
+    @FindBy(xpath = "//select[@id='checkout_shipping_address_province']")
+    private WebElement userDropDownState;
+
+    @FindBy(xpath = "//input[@id='checkout_shipping_address_zip']")
+    private WebElement userZipCode ;
+
+    @FindBy(xpath = "//input[@id='checkout_shipping_address_phone']")
+    private WebElement userPhone ;
+
+    @FindBy(xpath = "//button[@id='continue_button']")
+    private WebElement continueToShippingButton;
+
+    public WebElement setContinueToShipping(WebElement continueToShippingButton){
+        this.continueToShippingButton=continueToShippingButton;
+        return continueToShippingButton;
+    }
+
+//    public String setLoginApplication(String email, String firstName, String lastName, String company, String address1, String address2, String city, String stateDropdown, String zipCode, String phone) {
+//        this.userEmail= email;
+//        userFirstName.sendKeys(firstName);
+//        userLastName.sendKeys(lastName);
+//        userCompany.sendKeys(company);
+//        userAddress1.sendKeys(address1);
+//        userAddress2.sendKeys(address2);
+//        userCity.sendKeys(city);
+//
+//        //Dropdown element gelecek.dropdownu buraya koymak mantikli mi
+//        Select stateDropDown = new Select(userDropDownElement);
+//        stateDropDown.selectByVisibleText(stateDropdown);
+//
+//        userZipCode.sendKeys(zipCode);
+//        userPhone.sendKeys(phone);
+//    }
+
+    public void getLoginApplication(String email, String countryDropdown, String firstName, String lastName, String company, String address1, String address2, String city, String stateDropdown, String zipCode, String phone) {
+        userEmail.sendKeys(email);
+        Select countryDropDown = new Select(userDropDownCountry);
+        countryDropDown.selectByVisibleText(countryDropdown);
+        userFirstName.sendKeys(firstName);
+        userLastName.sendKeys(lastName);
+        userCompany.sendKeys(company);
+        userAddress1.sendKeys(address1);
+        userAddress2.sendKeys(address2);
+        userCity.sendKeys(city);
+        Select stateDropDown = new Select(userDropDownState);
+        stateDropDown.selectByVisibleText(stateDropdown);
+        userZipCode.sendKeys(zipCode);
+        userPhone.sendKeys(phone);
+    }
+
+    public void getContinueToShipping(){
```

Comment:
shipping word is an action "getContinueToShippingPageButton" would be more understandable. please apply to all getters.
https://thecoderoad.blog/2020/03/29/clean-code-naming-conventions/#:~:text=Class%20and%20Method,use%20your%20code.

Reply:
I change like this "getContinueToShippingPage" because I felt more right.

Reply:
I did.